src/plugins/data_source_management/framework/utils/shared.ts

```diff
@@ -70,6 +70,7 @@ export const EDIT = '/edit';
 export const DATACONNECTIONS_UPDATE_STATUS = '/status';
 export const SECURITY_ROLES = '/api/v1/configuration/roles';
 export const EVENT_ANALYTICS = '/event_analytics';
+export const REMOTE_CLUSTER = '/remote_cluster';
```

Comment:
unrelated to this pr, but this looks like these arejust constants which should be top level to avoid redfining this everytime